app/controllers/sign_up/completions_controller.rb

```diff
@@ -89,6 +89,12 @@ def analytics_attributes(page_occurence)
         needs_completion_screen_reason: needs_completion_screen_reason,
       }
 
+      if current_user.in_person_enrollments.any?
+        last_enrollment = current_user.in_person_enrollments.last
+        attributes[:in_person_proofing_status] = last_enrollment.status
+        attributes[:doc_auth_result] = last_enrollment.doc_auth_result
```

Comment:
it'd be hepful to add _in_person_proofing_status_ and _doc_auth_result_ to _user_registration_complete_ method in analytics_events.rb ?